src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/BoundaryLevelRepository.cs

```diff
@@ -29,5 +53,23 @@ public IEnumerable<BoundaryLevel> FindByGeographicLevels(IEnumerable<GeographicL
                 .Where(level => geographicLevels.Contains(level.Level))
                 .OrderByDescending(level => level.Published);
         }
+
+        public async Task<BoundaryLevel> Create(
+            GeographicLevel level,
+            string label,
+            DateTime published)
+        {
+            var newBoundaryLevel = await _context.BoundaryLevel.AddAsync(new()
+            {
+                Level = level,
+                Label = label,
+                Created = DateTime.UtcNow,
```

Comment:
We haven't been setting any created values explicitly recently because we have interfaces in `Timestamps.cs` which can be used which when present mean the entities get `Created` or `Created` and `Updated` values set automatically.

It should be just a case of changing `BoundaryLevel` to `public class BoundaryLevel : ICreatedTimestamp<DateTime>`.

Since they can now be updated too when labels change you could consider using `ICreatedUpdatedTimestamp<DateTime,DateTime?>` . This will force you to add a `DateTime? Updated` property and then you will need to recreate your database migration to add the new column.